dashboard/config/routes.rb

```diff
@@ -229,11 +218,7 @@ def redirect_to_teacher_dashboard
   post '/admin/gatekeeper/set', :to => 'dynamic_config#gatekeeper_set', as: 'gatekeeper_set'
   get '/admin/:action', controller: 'reports', as: 'reports'
 
-  get '/stats/usage/:user_id', to: redirect_to_teacher_dashboard
-  get '/stats/students', to: redirect_to_teacher_dashboard
-  get '/stats/:user_id', to: redirect_to_teacher_dashboard
   get '/popup/stats', to: 'reports#header_stats', as: 'header_stats'
-  get '/redeemprizes', to: 'reports#prizes', as: 'my_prizes'
```

Comment:
Actually I don't think we can remove this yet.  There are still references to this route [in the user menu](https://github.com/code-dot-org/code-dot-org/blob/479ee2fb1803ae79e349ff0ccf2fcfbcfcfc0c40/dashboard/app/views/shared/_user_header.html.haml#L13-L14).

These prizes predate the 2014 Hour of Code.  Maybe we shouldn't remove them from the DB, but should we remove the UI for accessing them?  The prizes for the past two years have been issued from Pegasus.